check.go

```diff
@@ -29,6 +29,10 @@ Loop:
 		if !disableSkipCI && ContainsSkipCI(p.Tip.Message) {
 			continue
 		}
+		// Filter pull request if the BaseBranch does not match the one specified in source
+		if p.PullRequestObject.BaseRefName != request.Source.BaseBranch && len(request.Source.BaseBranch) > 0 {
```

Comment:
I think I would rewrite this a bit:
```go
if request.Source.BaseBranch != "" && p.PullRequestObject.BaseRefName != request.Source.BaseBranch {
  ...
}
```

- `request.Source.BaseBranch != ""` since it's a string.
- Check whether `BaseBranch` has been set before comparing with the PR, since in most cases the first condition will be false.